requirements/requirements.cogvlm.txt

```diff
@@ -1,4 +1,4 @@
-transformers>=4.36.0,<4.38.0
+transformers>=4.36.0
```

Comment:
I remember pinning the version due to: https://github.com/roboflow/inference/issues/355
Not sure if never versions of transformers solve the problem, but if yes, probably lower-bound should be bumped